osvr/RenderKit/RenderManagerOpenGLATW.h

```diff
@@ -652,7 +653,7 @@ namespace renderkit {
                     expectedFrameInterval = static_cast<double>(timing.hardwareDisplayInterval.seconds +
                                                                 timing.hardwareDisplayInterval.microseconds / 1e6);
                 } else {
-                    m_log->error() << "RenderManagerThread::threadFunc() = couldn't get timing info";
+                    //m_log->error() << "RenderManagerThread::threadFunc() = couldn't get timing info";
```

Comment:
If this is not desired, probably delete it rather than commenting it out.

Reply:
Will do.